cfconfiguration/cffinder_unix.go

```diff
@@ -0,0 +1,50 @@
+// +build !windows
+
+package cfconfiguration
+
+import "go/build"
+import "os"
+import "os/user"
+
+var checkFileExistence = func (name string) (os.FileInfo, error) { return os.Stat(name) }
+
+func getUserConfigFile() (val string, ok bool) {
+	const relativeUserConfigPath = "/.config/cftool/config.yaml"
+
+	usr, err := user.Current()
+	if err != nil {
+		return "", false
+	}
+
+	userConfigPath := usr.HomeDir + relativeUserConfigPath
+
+	if _, err := checkFileExistence(userConfigPath); err != nil {
```

Comment:
Let's remove assignments from `if` statements.

Reply:
Updated in cffinder_unix.go.